charts/incubator/jts3servermod/values.yaml

```diff
@@ -0,0 +1,42 @@
+image:
+  repository: ghcr.io/galexrt/jts3servermod
+  pullPolicy: IfNotPresent
+  tag: main@sha256:fb8fb98f77b3dc83c44bf41dcfa74cb7f6c45972a954f856d5cc0fe2c7282a85
+portal:
+  open:
+    enabled: false
+persistence:
+  config:
+    enabled: true
+    mountPath: "/jts3servermod/config"
+  logs:
+    enabled: true
+    mountPath: "/jts3servermod/logs"
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+securityContext:
+  container:
+    runAsNonRoot: true
+    readOnlyRootFilesystem: true
+    runAsUser: 568
+    runAsGroup: 568
```

Comment:
```suggestion
securityContext:
  container:
    readOnlyRootFilesystem: false
```

Reply:
Those are the settings he is using for his Helm chart in the repo. So I took those

Reply:
we default to to those ;) except the ROFS false